include/avif/avif.h

```diff
@@ -7,7 +7,16 @@
 #include <stddef.h>
 #include <stdint.h>
 
-#ifndef AVIF_API
+// ---------------------------------------------------------------------------
+// Export macros
+
+// AVIF_BUILDING_SHARED_LIBS should only be defined when building shared
+// version of libavif library.
```

Comment:
Nit: building shared version of libavif library => libavif is being built as a shared library ?